config/main.py

```diff
@@ -546,6 +546,15 @@ def warm_restart_neighsyncd_timer(ctx, seconds):
         ctx.fail("neighsyncd warm restart timer must be in range 1-9999")
     db.mod_entry('WARM_RESTART', 'swss', {'neighsyncd_timer': seconds})
 
+@warm_restart.command('bgp_timer')
+@click.argument('seconds', metavar='<seconds>', required=True, type=int)
+@click.pass_context
+def warm_restart_bgp_timer(ctx, seconds):
+    db = ctx.obj['db']
+    if seconds not in range(1,3600):
```

Comment:
Can you update https://github.com/Azure/sonic-swss/blob/master/doc/swss-schema.md file or use the value here that was specified in the doc?

Reply:
Yep, will update it. Btw, the reason i'm changing this value is to match FRR's graceful-restart range -- just for consistency.

Reply:
done